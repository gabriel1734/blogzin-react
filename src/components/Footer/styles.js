import styled, { css } from 'styled-components';

export const Container = styled.footer`
  ${({ theme }) => css`
    text-align  : center;
  span{
    color: ${theme.colors.secondaryColor};
  }
  a {
      color: ${theme.colors.primaryColor};
      text-decoration: none;
      font-size: ${theme.fonts.sizes.small};
    }
  `}
`;
