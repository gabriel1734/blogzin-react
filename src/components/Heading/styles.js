import styled, { css } from 'styled-components';

export const Title = styled.h1`
  ${({ theme }) => css`
    background-color: ${theme.colors.mainBg};
  `}`;
