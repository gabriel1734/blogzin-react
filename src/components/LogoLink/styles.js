import styled, { css } from 'styled-components';

export const Container = styled.a`
  ${({ theme }) => css`
    display: flex;
    align-items: center;
    color: ${theme.colors.primaryColor};
    text-decoration: none;
    

    > img {
      max-height: 3rem;
    }
  `}
`;
