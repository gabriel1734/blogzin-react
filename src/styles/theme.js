export const theme = {
  colors: {
    mainBg: 'pink',
    secondaryBg: 'blue',
  },
  fonts: {},
  spacings: {},
};
