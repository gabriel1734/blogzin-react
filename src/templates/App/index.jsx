import * as Styled from './styles';
function Home() {
  return (
    <Styled.Wrapper>
      <div>
        <h1>hello</h1>
      </div>
    </Styled.Wrapper>
  );
}

export default Home;
