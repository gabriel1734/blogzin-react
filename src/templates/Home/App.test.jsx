import App from '.';
import { renderTheme } from '../../styles/render-theme';

describe('App', () => {
  it('should render correctly', () => {
    const { container } = renderTheme(<App />);
    expect(container).toMatchSnapshot();
  });
});
