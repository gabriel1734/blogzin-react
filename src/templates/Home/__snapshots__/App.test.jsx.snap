// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
.c0 {
  position: absolute;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  z-index: 10;
  background: #0A1128;
}

.c0::before,
.c0::after {
  content: '';
  position: absolute;
  top: 50%;
  left: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  border-width: .5rem;
  border-color: transparent;
  border-style: solid;
  border-radius: 50%;
}

.c0::after {
  width: 6rem;
  height: 6rem;
  border-left: .5rem solid #dc143c;
  border-top: .5rem solid #dc143c;
  -webkit-animation: hOOCWl .6s linear infinite;
  animation: hOOCWl .6s linear infinite;
}

.c0::before {
  width: 2rem;
  height: 2rem;
  border-left: .5rem solid #dc143c;
  border-top: .5rem solid #dc143c;
  -webkit-animation: hOOCWl 1s linear infinite reverse;
  animation: hOOCWl 1s linear infinite reverse;
}

<div>
  <div
    class="c0"
  />
</div>
`;
